src/components/table/OcTableFiles.vue

```diff
@@ -679,8 +658,6 @@ export default {
   <oc-table-files :resources="resources" :arePathsDisplayed="true" v-model="selected" />
 </template>
 <script>
-  import { shareType } from "../../utils/shareType"
```

Comment:
This also is on (our) current master and looks like you're running an outdated version. Please rebase on our master and apply your changes there